src/fzx/tui/color.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+#include "fzx/macros.hpp"
+#include <cstdint>
+#include <sstream>
+#include <variant>
+#include <fmt/core.h>
+
+namespace parse_hex {
```

Comment:
Wrap the entire file in `namespace fzx`. `parseNibble` and `parseByte` doesn't have to be in a separate namespace

Reply:
i wanted to hide it

Reply:
done